_data/events_gdcr2018/Pune-CDKGlobal.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Coderetreat @ CDK Global (India) Pvt. Ltd.",
+  "url": "https://sites.google.com/view/cdkpuneevents",
+  "moderators": [
+    "Vibhor.Mamgain@cdk.com",
```

Comment:
Preferrably, we would like to see your twitter handles here. If not possible, your full name should also be fine. There is no need to share your email with public :-)